### PR TITLE
Create 0001-dont-use-olm.md

### PR DESCRIPTION
**Pull request by Gkrumbach07**
Add OLM ADR

related: https://github.com/open-services-group/devsecops/issues/10

**TODO**
- [x] naming convention for ADRs?

### CODE REVIEW

File: ADR/0001-dont-use-olm.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# How to deploy ODH
+
+- Status: acepted
+- Deciders: Harshad, Humair, Gage, Gregory
```
Comment (Gregory-Pereira) [inline ADR/0001-dont-use-olm.md]: I think Anand is included in this too

Comment (HumairAK) [reply ADR/0001-dont-use-olm.md]: done

**Comment by HumairAK** [comment]
@Gregory-Pereira / @harshad16 I see there's a build happening in ci, why?

**Comment by harshad16** [comment]
> I see there's a build happening in ci, why?

@HumairAK  that happens, when pr is not rebased, as it is looking for the pr config in the pr branch.
please rebase and it would go away.

**Pull request merged**